### ISSUE TITLE
Use with ZenHub

### ISSUE THREAD

**Issue by Mottie** [buunguyen/octotree#242]
Zenhub added a new button to expand and collapse their board view tab

![2016-01-22 19_15_48-boards zenhubio_support](https://cloud.githubusercontent.com/assets/136959/12527192/cb5b536e-c13c-11e5-9c78-6bcf5477632e.png)

The problem is when the boards are expanded, the Octotree button overlaps the toggle button

![2016-01-22 19_15_02-boards zenhubio_support](https://cloud.githubusercontent.com/assets/136959/12527198/e2d7ffec-c13c-11e5-9269-a47e1755cfb1.png)

I would like to ask for an enhancement to Octotree where a class name is added to the body, so that I might include the following css to shift ZenHub's buttons to the right, but only when a user has the Octotree extension.

```css
body.octotree .zh-board-menu .tabnav-left {
	margin-left: 20px;
}
```

Of course, the body class name can be of your choosing, but once the css selector is added, we can apply our GitHub-Dark style and end up with this:

![2016-01-22 19_21_26-boards zenhubio_support](https://cloud.githubusercontent.com/assets/136959/12527246/815b171c-c13d-11e5-94dd-c7e1aa0268ce.png)

Thanks!

**Comment by buunguyen** [comment]
See https://github.com/buunguyen/octotree/commit/5d5f93e88754167acb27553bbecd3c081e28bb78.

Basically octotree will add "octotree" class to html and "octotree-show" class only when the sidebar is displayed.

**Comment by buunguyen** [comment]
Sorry, close by mistake. Before I release to the store, can you verify if this addresses the issue at hand?

**Comment by Mottie** [comment]
Hmm, I can't really test it. Every time I add "octotree" class to the HTML element, the side bar opens. When I close it the class disappears.

**Comment by Mottie** [comment]
Tested with slightly altered class names and it looks like it'll work as expected. Thanks!

**Comment by buunguyen** [comment]
Released in 2.0.8.

**Comment by Mottie** [comment]
Works great, thanks again!
